bootstrap_cfn/stacks/iam-deploy.json

```diff
@@ -0,0 +1,74 @@
+{
+    "CrossAccountRole" : {
+        "Type" : "AWS::IAM::Role",
+        "Properties" : {
+            "AssumeRolePolicyDocument" : {
+                "Statement" : [
+                     {
+                         "Effect" : "Allow",
+                         "Principal" : {
+                             "AWS": "arn:aws:iam::ACCOUNT_NUMBER_HERE:root"
```

Comment:
I think you just specify the parameters in the API call to CreateStack. Checking